packages/common/src/Adaptor.js

```diff
@@ -481,6 +481,16 @@ export function humanProper(str) {
   }
 }
 
+/**
+ * Splits an object into two objects based on a list of keys.
+ * The first object contains the keys that are not in the list,
+ * and the second contains the keys that are.
+ * @public
+ * @function
+ * @param {Object} obj - The object to split.
+ * @param {string[]} keys - List of keys to split on.
+ * @returns {Object[]} - Tuple of objects, first object contains keys not in list, second contains keys that are.
+ */
 export function splitKeys(obj, keys) {
```

Comment:
Is it correct that we're publicly documenting  `splitKeys` now?

I'm not against it but it doesn't feel related to this work. It should really be mentioned in the main PR description.

Reply:
This was me, I guess we were exporting it and _not_ documenting it - so I thought we should document the interface.

@mtuchi can you update the PR notes to include something like "Added documentation for `splitKeys`.